lib/checks/aria/aria-required-children-evaluate.js

```diff
@@ -42,17 +42,6 @@ function missingRequiredChildren(virtualNode, role, required, ownedRoles) {
 
   // combobox exceptions
   if (isCombobox) {
```

Comment:
All of this special case should probably be removed. We don't want to check for required owned elements.